versioned_docs/version-2.x/magento1/search-engine.mdx

```diff
@@ -253,3 +253,402 @@ Front-Commerce 2.13.
+```
+
+:::info
+
+You can find your search API key by logging in in into
+[the XO console](https://console.early-birds.io/), and navigating to:
+`Search > API Keys`
+
+:::
+
+The script should be very quick to execute, and will let you now what needs your
+attention in your Attraqt index. If you have any doubt on how to change this, do
+not hesitate to ask your Attraqt CSM.
+
+When you get the ✔️ as a response from the script, you are ready to use Attraqt
+with Front-Commerce.
+
+### Facets
+
+Due to facet IDs not being editable in Attraqt, facets created in your Attraqt
+console must be based on your Magento attributes exact names.
+
+By example, if you have an attribute `fashion_colour` in your product and you
+want to add a facet for it, the facet in attraqt _must_ be based on an attribute
+named `fashion_colour` as well, which will create a facet with ID
+`facet-fashion_colour`.
+
+### Front-Commerce configuration
+
+First, you need to make sure the Attraqt's search client is installed:
+
+```shell
+npm i @attraqt/xo-js@1.6.0
+```
+
+On Front-Commerce side, you need to enable the Attraqt datasource by making the
+following changes in your .front-commerce.js file:
+
+```js title=".front-commerce.js"
+module.exports = {
+  // highlight-next-line
+  modules: ["./node_modules/front-commerce/modules/datasource-attraqt"],
+  serverModules: [
+    { name: "FrontCommerce", path: "server/modules/front-commerce" },
+    // highlight-start
+    {
+      name: "Magento1Attraqt",
+      path: "datasource-attraqt/server/modules/magento1-attraqt",
+    },
+    // highlight-end
+    { name: "Magento1", path: "server/modules/magento1" },
+  ],
+};
+```
+
+:::caution Known issue
+
+The Attraqt server module needs to be enabled **before** the Magento module.
+
+:::
+
+Then, in your `.env` file, you need to define the
+`FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY` environment variable:
+
+```bash title=.env
+FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY=yourapikeyvalue
+```
+
+After restarting Front-Commerce, you should be able run a GraphQL query to
+search for products, categories and/or pages, for instance:
+
+```graphql title="http://localhost:4000/playground"
+query Search {
+  search(query: "whatever you want to search for") {
+    query
+    products(params: { from: 0, size: 5 }) {
+      total
+      products {
+        sku
+        name
+      }
+    }
+    categories(size: 5) {
+      name
+    }
+    pages(size: 5) {
+      title
+    }
+  }
+}
+```
+
+If you are using the default theme or the Chocolatine theme, the search bar
+should now be visible.
+
+:::tip
+
+In case of emergency, Front-Commerce provides
+[a configuration to quickly deactivate Attraqt's search feature](/docs/2.x/reference/environment-variables#attraqt).
+Use the `FRONT_COMMERCE_ATTRAQT_DISABLED=true` and restart your application to
+temporarily deactivate the module.
+
+:::
+
+### Add contextual information to your queries
+
+<SinceVersion tag="2.23" />
```

Comment:
```suggestion
<SinceVersion tag="2.27" />
```

Reply:
I completely removed this one since the whole `Attraqt` section is already scoped in the 2.27.